src/resources/classes/controller.js

```diff
@@ -5,21 +5,23 @@ const getAll = async (req, res) => {
         const getAllClasses = await prisma.class.findMany({
         });
 
-        res.json({ data: getAllClasses });
+        res.json(getAllClasses);
     } catch (error) {
         console.error({ error })
         res.json({ error })
     }
 }
 
 const createOne = async (req, res) => {
+
+    console.log("body date: ", req.body.classStartDate , "\n body: ", req.body)
     try {
         const createOneClass = await prisma.class.create({
             data: {
                 className: req.body.className,
                 classType: req.body.classType,
                 classStatus: req.body.classStatus,
-                classStartDate: new Date(req.body.classStartDate),
+                classStartDate: new Date(req.body.classStartDate).toISOString(),
```

Comment:
If `classStartDate` has a type of `DateTime` in the schema, you shouldn't need to do `.toISOString()` - as long as `req.body.classStartDate` is a valid date string (e.g. "2021-01-01"), `classStartDate: new Date(req.body.classStartDate)` should be fine

Reply:
the issue occurs when the front end DATETIME does not match the `schema.prisma` one
`2021-11-22T00:00:00.000Z`
post is successful with `200` status
but the console logs out the body empty
`body date:  undefined `
 `body:  {}`
 if I don't use ISO, ill get smth like:
 `Sat Nov 13 2021 13:20:00 GMT+0200 (Eastern European Standard Time)`
 🤔